src/master-playlist-controller.js

```diff
@@ -369,10 +327,20 @@ export default class MasterPlaylistController extends videojs.EventTarget {
     }
   }
 
-  useAudio(label) {
-    let audioEntry;
-    let newAudioPlaylistLoader;
-    let mediaGroupName = this.masterPlaylistLoader_.media().attributes.AUDIO;
+  useAudio() {
+    let media = this.masterPlaylistLoader_.media();
+    let master = this.masterPlaylistLoader_.master;
+
+    if (!media || !media.attributes || !media.attributes.AUDIO ||
```

Comment:
I think in this case it may be better for us to not do some guards. They are definitely safe to have, but it might hide a bug later on (for instance, if someone does try setting useAudio before they are supposed to, we'd simply return here, instead of an error being thrown which could clue someone into the problem--that they are calling useAudio too early, before they are supposed to--immediately).